.github/workflows/main.yml

```diff
@@ -37,22 +37,23 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install dependencies
-        run: >-
-          sudo apt install -y
-          build-essential
-          php7.4-cli
-          haskell-platform
-          fp-compiler
-          python2.7
-          cppreference-doc-en-html
-          cgroup-lite
-          libcap-dev
-          python-dev
-          libpq-dev
-          libcups2-dev
-          libyaml-dev
-          rustc
-          mono-mcs
+        run: |
+          sudo apt update -y
+          sudo apt install -y \
```

Comment:
You can still use the folded style (`>`) with multiple commands: separate the commands with an empty line.

Reply:
Wow, YAML has a bit too many string variants 😅

It's just a pity that

```yaml
run: >
  sudo apt-get update -y

  sudo apt-get install
    aaa
    bbb
    ccc
```

becomes `"sudo apt-get update -y\nsudo apt-get install\n  aaa\n  bbb\n  ccc"`.
I'm not a fan of not indenting the packages a bit, and I think that with the backslashes (while a bit more verbose) the command is much more clear (at least to the ones not that expert in YAML). What do you think?

Reply:
Hmm, I understand the reasoning, although `\` feels like it reduces the clarity, too. You could actually use the plain flow style:
```yaml
run:
  sudo apt-get update
  
  sudo apt-get install
    aaa
    bbb
    ccc
```
How does this variant look?

Reply:
The more I look at YAML, the less I feel to know about it.

For sure this looks the cleanest, but when I look at those CI steps I imagine them as "bash scripts" that are executed one after the other. Without `\` and with just YAML magic I felt a bit confused on how they should be read. This is the cleanest variant but the furthest to a "bash script", and I feel like that installing packages should be closer to bash territory.

In any case, I don't think this is super important and mine is not a strong opinion, either way is fine for me. I'll leave to you the final decision.

Reply:
Alright, let's leave it like that.